src/gwt/src/org/rstudio/core/client/Version.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Version.java
+ *
+ * Copyright (C) 2020 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.core.client;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// A helper class for parsing numeric versions.
+public class Version
+{
+   public static final int compare(String lhs, String rhs)
```

Comment:
I think the idiomatic way of doing this in Java is to override `compareTo` for your class.